pipeline_steps/release.groovy

```diff
@@ -0,0 +1,24 @@
+void tag(String version, String ref){
+  print "Tagging ${version} at ${ref}"
+  sshagent (credentials:['rpc-jenkins-svc-github-ssh-key']){
+    sh """/bin/bash -xe
+      git fetch --all --tags
+      git tag -a -m \"${version}\" \"${version}\" origin/${ref}
+      git push --tags origin
```

Comment:
I'd prefer to see this only push the tag that has just been created - `git push origin \"${version}\"`. While it shouldn't make any difference, I'd prefer to exclude the possibility of the unexpected.

Reply:
Done